mne/utils/docs.py

```diff
@@ -2384,20 +2384,22 @@
     .. note:: This parameter controls the time period used in conjunction with
               both, ``reject`` and ``flat``.
 """
-docdict['epochs_events_event_id'] = """
+docdict['events'] = """
 events : array of int, shape (n_events, 3)
-    The events typically returned by the read_events function.
-    If some events don't match the events of interest as specified
-    by event_id, they will be marked as 'IGNORED' in the drop log.
+    The array of :term:`events`. The first column contains the event time in
+    samples, with :term:`first_samp` included. The third column contains the
+    event id."""
+docdict['events_epochs'] = """%s
+    If some events don’t match the events of interest as specified by event_id,
+    they will be marked as ‘IGNORED’ in the drop log.""" % docdict['events']
```

Comment:
probably not a big deal, but these line contain smart quotes... I think we don't usually include those in our docstrings.

Reply:
Copy-paste from the old docstring, but you are right, I'll replace the quotes with double backquotes.